cms/djangoapps/contentstore/models.py

```diff
@@ -293,7 +293,7 @@ def __str__(self):
     @classmethod
     def update_or_create(
         cls,
-        upstream_container: Container | None,
+        upstream_container: int | None,
```

Comment:
Is this working? I would think that below on line 322 you need to set `upstream_container_pk` to this value then, not `upstream_container`. But maybe the Django ORM is sorting it out?

Reply:
Ah sorry, I had that on my list to fix before pushing, but forgot.
I've fixed it, but I would have expected tests to fail without this change?

Reply:
I remember seeing a similar thing before where assigning an ID to an object field still worked silently. In any case while I'm testing this I'll make sure the `ContainerLink` is getting created.